Wolox/frontend-bootstrap
# Now uglifying/minifying only in Production

## Summary
Gulp Build now uglifying/minifying only in Production environment.

## Test Plan
1) Run `npm install` and `bower update`.
2) Run `gulp development`.
3) Open the [server](http://localhost:3000). It should be working normally. The CSS and JS should NOT be minified/uglified.
4) Run `gulp production`.
5) Open the [server](http://localhost:3000). It should be working normally. The CSS and JS SHOULD be minified/uglified.
